app/src/renderer/components/govern/ChartVotes.vue

```diff
@@ -57,10 +61,16 @@ export default {
             borderWidth: 0,
             data: this.chartValues,
             backgroundColor: [
-              'hsl(210,41%,49%)',
-              'hsl(60,18%,70%)',
-              'hsl(345,75%,49%)',
-              'hsl(210,32%,12%)'
+              'hsl(0,0%,100%)',
+              'hsl(233,96%,60%)',
+              'hsl(326,96%,59%)',
+              'hsl(233,13%,50%)'
+              /*
```

Comment:
what's with these?

Reply:
`ChartVotes` is used in the `LiProposal` component -- it's the CircleGraph. The graph colors don't seem to be editable through CSS so I hardcoded them in here.